standard/coordinates.md

```diff
@@ -0,0 +1,412 @@
+straight line elements the entrance and exit frames are colinear. B) For bends elements, the 
+transformation from entrance to exit coordinates is a rotation about the bend center.
+C) For `Patch` and `floor_shift` elements the 
+exit frame may be arbitrarily positioned with respect to the entrance frame.
+```
+
+%-----------------------------------------------------------------------------
+
+As discussed in the next section, the branch coordinate system is constructed starting with the first
+element in a lattice tracking branch and then building up the coordinate system element-by-element.
+Most elements in a tracking branch have an "`entrance`" and an "`exit`" coordinate frame as
+illustrated in the above figure.
+These coordinate frames are attached to the element and are part of the `element body coordinates`. 
+`Fiducial` elements ([xxx](#s:fiducial)) are an excption. 
+`Fiducial` elements only have a single coordinate frame that is tied to global coordinates 
+and construction of the branch coordinate system starts at this coordinate system. 
+See [xxx](#s:fiducial) for more details.
+Note that `Girder` elements ([xxx](#s:girder)) also only have a single coordinate frame but they
+are put in a lord branch and do not appear in a tracking branch.
+
+Most element types have a "straight" geometry as shown in
+[](#f:ele.coord.frame)A. That is, the reference curve through the element is a straight line
+segment with the {math}`x` and {math}`y` axes always pointing in the same direction.
+For a [Bend](#s:bend}) element the reference curve is a segment of a circular arc as shown in
+[](#f:ele.coord.frame)B. With the `ref_tilt` parameter of a bend set to zero, the rotation axis
+between the entrance and exit frames is parallel to the {math}`y`-axis ([xxx](#s:floor})).
+For [Patch](#s:patch}) and [floor_shift](#s:floorshift)
+elements ([](#f:ele.coord.frame)C), the exit face can be
+arbitrarily oriented with respect to the entrance end.
+For `FloorShift` elements the interior reference curve between the
+entrance and exit faces is not defined. For the `Patch` element, the interior reference curve 
+is dependent upon certain `Patch` element parameter settings ([xxx](#s:patch)) and, in general,
+will have a discontinuity.
+
+%-----------------------------------------------------------------------------
+(s:ref.construct)=
+## Branch Coordinates Construction
+
+%-----------------------------------------------------------------------------
+
+```{figure} figures/patch-between.svg
+:width: 70%
+:label: f:patch.between
+
+A) The branch coordinates are constructed by
+connecting the `downstream` reference frame of one element with the `upstream` reference frame
+of the next element in the branch. Coordinates shown is for the mating of element `A` to element
+`B`.  B) Example with drift element `dft` followed by a bend `bnd`. Both elements are
+unreversed. C) Similar to (B) but in this case element `bnd` is reversed.  D) Similar to (C) but
```

Comment:
Not sure it makes sense to use the word "unreversed".  I think it could be removed.

Reply:
Reversing elements is explained in the text body and is appropriate here.